pkg/apis/admission/types.go

```diff
@@ -56,6 +56,27 @@ type AdmissionRequest struct {
 	// "binding", and kind "Binding".
 	// +optional
 	SubResource string
+
+	// RequestKind is the type of object being manipulated by the the original API request.  For example: Pod
+	// If this differs from the value in "kind", an equivalent match and conversion was performed.
+	// See documentation for the "matchPolicy" field in the webhook configuration type.
+	// +optional
+	RequestKind *metav1.GroupVersionKind
+	// RequestResource is the name of the resource being requested by the the original API request.  This is not the kind.  For example: pods
+	// If this differs from the value in "resource", an equivalent match and conversion was performed.
+	// See documentation for the "matchPolicy" field in the webhook configuration type.
+	// +optional
+	RequestResource *metav1.GroupVersionResource
+	// RequestSubResource is the name of the subresource being requested by the the original API request.  This is a different resource, scoped to the parent
+	// resource, but it may have a different kind. For instance, /pods has the resource "pods" and the kind "Pod", while
+	// /pods/foo/status has the resource "pods", the sub resource "status", and the kind "Pod" (because status operates on
```

Comment:
odd to get an example where the kind is the same. Expected the opposite, e.g. for scale after "for instance".

Reply:
yeah, most of this was copied/pasted from the existing SubResource doc. will update both to be better